doc/architectural/front-page.md

```diff
@@ -1,5 +1,5 @@
-CProver Developer Documentation
-=====================
+
```

Comment:
Nit pick: blank line at the top of the file?

Reply:
Done.